test/browser/render.js

```diff
@@ -911,5 +911,43 @@ describe('Rendering', function () {
         }
       });
     });
+
+    it('should ignore newer values of old nested observables', function (done) {
```

Comment:
all but the latest one :P
If a parent Observable A emits 3 child observables X, Y and Z. X and Y are the old ones, Z is the latest one. We only want the values emitted by Z.
But yeah, there might be a better name describing it.